AppDB/appscale/datastore/scripts/ua_server_backup.py

```diff
@@ -0,0 +1,134 @@
+""" UA Server data backup. """
+
+import argparse
+import errno
+import logging
+import os
+import time
+
+from tornado import gen
+
+from appscale.common.constants import LOG_FORMAT
+from appscale.datastore import appscale_datastore
+from appscale.datastore.dbconstants import (
+  AppScaleDBConnectionError,
+  USERS_SCHEMA,
+  USERS_TABLE
+)
+from appscale.datastore.utils import tornado_synchronous
+
+# The location where the backup files will be stored.
+BACKUP_FILE_LOCATION = "/opt/appscale/backups/"
+
+
+@tornado_synchronous
+@gen.coroutine
+def get_table_sync(datastore, table_name, schema):
+  """ Gets data from datastore.
```

Comment:
It actually gets data from Cassandra. Though it uses datastore code for doing it.
Though you had a reason to name `datastore` this way (this object has type `DatastoreProxy` I think) it's confusing. This object is rather Cassandra adapter..